### ISSUE TITLE
Randomized pin entry

### ISSUE THREAD

**Issue by michaelni** [mooltipass/minible#371]

**Comment by limpkin** [comment]
I must be missing something... but moolticute has an option allowing you to set a random starting pin...

**Comment by michaelni** [comment]
> I must be missing something... but moolticute has an option allowing you to set a random starting pin...

Indeed, sorry i have missed that. I wonder why its not default though
Thanks
